native/package.json

```diff
@@ -11,13 +11,13 @@
     "start:aschaffenburg": "cross-env BUILD_CONFIG_NAME='aschaffenburg' react-native start --reset-cache",
     "start:integreat-e2e": "cross-env BUILD_CONFIG_NAME='integreat-e2e' react-native start --reset-cache",
     "android": "yarn android:integreat-test-cms",
-    "android:integreat-test-cms": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' react-native run-android --no-packager",
-    "android:malte": "cross-env BUILD_CONFIG_NAME='malte' react-native run-android --no-packager",
-    "android:aschaffenburg": "cross-env BUILD_CONFIG_NAME='aschaffenburg' react-native run-android --no-packager",
+    "android:integreat-test-cms": "cross-env BUILD_CONFIG_NAME='integreat-test-cms' react-native run-android --no-packager --appId app.integreat.test",
+    "android:malte": "cross-env BUILD_CONFIG_NAME='malte' react-native run-android --no-packager --appId de.malteapp",
+    "android:aschaffenburg": "cross-env BUILD_CONFIG_NAME='aschaffenburg' react-native run-android --no-packager --appId app.aschaffenburg",
     "android:integreat-e2e": "cross-env BUILD_CONFIG_NAME='integreat-e2e' react-native run-android --no-packager",
     "android:integreat:production": "cross-env BUILD_CONFIG_NAME='integreat' react-native run-android --no-packager --variant=release",
```

Comment:
no `appId` here?

Reply:
no, as the default is fine with the production